runtime/port/zos390/j9guardedstorage.c

```diff
@@ -53,8 +53,8 @@
 
 __asm(" IEAGSF PLISTVER=MAX,MF=(L,SPARAM)": "DS"(sParam));
 
-#pragma prolog(enable_guarded_storage_facility,"MYPROLOG")
-#pragma epilog(enable_guarded_storage_facility,"MYEPILOG")
+#pragma prolog(enable_guarded_storage_facility,main)
+#pragma epilog(enable_guarded_storage_facility,main)
```

Comment:
Why did we need to change from `MYPROLOG/MYEPILOG` to `main`?   What happens if we don't make that change?   From the [MetalC KC page](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.1.0/com.ibm.zos.v2r1.ccrug00/prologcode.htm), with `main`:

>  "a STORAGE macro is used to obtain and release a dynamic storage area of 1 MB. For functions other than "main", the prolog code simply picks up its DSA pointer (the NAB pointer) from the "Address of next save area" field in the calling function’s save area."

Do we really need that 1MB of DSA?

Reply:
Basically because the implementations of MYPROLOG/MYEPILOG are not in OpenJ9. Seems we should include them.